# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/test/single-xing-header-no-content-length.mp3^headers^
# Commit: f33e1ce6dcf1
# Full Hash: f33e1ce6dcf19e47952f3963c8e970c7c380acd2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-20 21:19:23
# Regressor Bug: 1843003
# File Overlap Count: 2
# Description:
#   Bug 1843003 - Test that typical mp3 broadcast work well. r=alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183737
# ==============================================================================

diff -r 763b34de3c14 -r f33e1ce6dcf1 dom/media/test/single-xing-header-no-content-length.mp3^headers^
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/media/test/single-xing-header-no-content-length.mp3^headers^	Thu Jul 20 13:17:59 2023 +0000
@@ -0,0 +1,3 @@
+HTTP 200 OK
+Content-Length: invalid
+Cache-Control: no-store